.github/workflows/docker-build.yml

```
@@ -0,0 +1,126 @@
+
+env:
+  REGISTRY: docker.io
+  BACKEND_IMAGE: monthly-expense-backend
+  FRONTEND_IMAGE: monthly-expense-frontend
+
+jobs:
+  docker-build-and-test:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout code
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)